sources/application/WebPage/WebPage.php

```diff
@@ -851,6 +915,55 @@ public function add_linked_stylesheet($s_linked_stylesheet, $s_condition = "")
 		$this->a_linked_stylesheets[$s_linked_stylesheet] = array('link' => $s_linked_stylesheet, 'condition' => $s_condition);
 	}
 
+	/**
+	 * Initialize compatibility linked stylesheets for the page
+	 *
+	 * @see static::COMPATIBILITY_MOVED_LINKED_STYLESHEETS_REL_PATH
+	 * @see static::COMPATIBILITY_DEPRECATED_LINKED_STYLESHEETS_REL_PATH
+	 * @throws \ConfigException
+	 * @throws \CoreException
+	 * @since 3.0.0
+	 */
+	protected function InitializeCompatibilityLinkedStylesheets(): void
+	{
+		$bIncludeDeprecatedFiles = utils::GetConfig()->Get('compatibility.include_deprecated_css_files');
+		if ($bIncludeDeprecatedFiles) {
+			$this->AddCompatibilityFiles(static::ENUM_COMPATIBILITY_FILE_TYPE_CSS, static::ENUM_COMPATIBILITY_MODE_DEPRECATED_FILES);
+		}
+
+		$bIncludeMovedFiles = utils::GetConfig()->Get('compatibility.include_moved_css_files');
+		if ($bIncludeMovedFiles) {
+			$this->AddCompatibilityFiles(static::ENUM_COMPATIBILITY_FILE_TYPE_CSS, static::ENUM_COMPATIBILITY_MODE_MOVED_FILES);
+		}
+	}
+
+	/**
+	 * Add compatibility files of $sFileType from $sMode (which are declared in {@see static::COMPATIBILITY_<MODE>_LINKED_<TYPE>_REL_PATH}) back to the page
+	 *
+	 * @param string $sFileType {@see static::ENUM_COMPATIBILITY_FILE_TYPE_XXX}
+	 * @param string $sMode {@see static::ENUM_COMPATIBILITY_MODE_XXX}
+	 *
```

Comment:
Maybe add 2 `@uses` to indicate which contants and methods are called ?

Reply:
Not sure to understand, you want me to replace the `@see` or to replace the stated constants? I did not put all the declinaisons of the constants on purpose to avoid flooding.

Reply:
Sorry, I should have wrote a suggestion
Here it is !

```suggestion
	 *
	 * @uses COMPATIBILITY_*_LINKED_STYLESHEETS
	 * @uses COMPATIBILITY_*_LINKED_SCRIPTS
	 * @uses add_linked_add_linked_stylesheet
	 * @uses add_linked_add_linked_script
	 *
```

Reply:
Done. After our (IRL) discussion I moved the comment to the `@uses add_linked_*` lines as the $sFileType determines the method called but is not used as-is in the method name.

Reply:
👌